distributed/tests/test_steal.py

```diff
@@ -1350,3 +1374,341 @@ def test_steal_worker_state(ws_with_running_task):
     assert "x" not in ws.tasks
     assert "x" not in ws.data
     assert ws.available_resources == {"R": 1}
+
+
+def test_balance_willing_to_move_costly_items(recompute_saturation):
+    """See also test_balance"""
```

Comment:
This and the following test replicate test cases from `test_balance`. This was initially done to ensure that the setup logic works correctly, we can probably remove those.